### PR TITLE
Removed Load/Save Race buttons from empire setup form.

### PR DESCRIPTION
**Pull request by unknown555**
Resolved vehicle designer warnings from being cut off by changing associated control from ListBox to TextBox, which allows multiline word wrapping.

### CODE REVIEW

File: FrEee.UI.WinForms/Forms/EmpireSetupForm.cs [hunk 1]
```diff
@@ -204,18 +204,18 @@ private void btnCompareCultures_Click(object sender, EventArgs e)
...
+	//	var dlg = new OpenFileDialog();
+	//	dlg.InitialDirectory = Path.Combine(Path.GetDirectoryName(Assembly.GetEntryAssembly().Location), "Races");
+	//	dlg.Filter = "Races (*.rac)|*.rac";
+	//	var result = dlg.ShowDialog();
+	//	if (result == DialogResult.OK)
+	//	{
+	//		EmpireTemplate.PrimaryRace = Race.Load(dlg.FileName);
+	//		Bind();
+	//	}
+	//}
```
Comment (ekolis) [inline FrEee.UI.WinForms/Forms/EmpireSetupForm.cs:218]: Might as well delete this function if it's no longer being used.

File: FrEee.UI.WinForms/Forms/EmpireSetupForm.cs [hunk 2]
```diff
@@ -232,16 +232,16 @@ private void btnOK_Click(object sender, EventArgs e)
...
+	//private void btnSaveRace_Click(object sender, EventArgs e)
+	//{
+	//	SaveChanges();
+	//	var dlg = new SaveFileDialog();
+	//	dlg.InitialDirectory = Path.Combine(Path.GetDirectoryName(Assembly.GetEntryAssembly().Location), "Races");
+	//	dlg.Filter = "Races (*.rac)|*.rac";
+	//	var result = dlg.ShowDialog();
+	//	if (result == DialogResult.OK)
+	//		EmpireTemplate.PrimaryRace.Save(dlg.FileName);
+	//}
```
Comment (ekolis) [inline FrEee.UI.WinForms/Forms/EmpireSetupForm.cs:244]: Might as well delete this one too.

File: FrEee.UI.WinForms/Forms/EmpireSetupForm.resx [hunk 3]
```diff
@@ -117,4 +117,35 @@
...
+        iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAABGdBTUEAALGPC/xhBQAAAAlwSFlzAAAO
+        wgAADsIBFShKgAAAAA1JREFUGFdjYGBgYAAAAAUAAYoz4wAAAAAASUVORK5CYII=
+</value>
+  </data>
+  <data name="picLeaderPortrait.Image" type="System.Drawing.Bitmap, System.Drawing" mimetype="application/x-microsoft.net.object.bytearray.base64">
+    <value>
+        iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAABGdBTUEAALGPC/xhBQAAAAlwSFlzAAAO
+        wgAADsIBFShKgAAAAA1JREFUGFdjYGBgYAAAAAUAAYoz4wAAAAAASUVORK5CYII=
+</value>
+  </data>
```
Comment (ekolis) [inline FrEee.UI.WinForms/Forms/EmpireSetupForm.resx:150]: Where did this come from? Do these picture boxes have default images now?

File: FrEee.UI.WinForms/Forms/VehicleDesignForm.resx [hunk 4]
```diff
@@ -117,4 +117,17 @@
...
+        iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAABGdBTUEAALGPC/xhBQAAAAlwSFlzAAAO
+        wAAADsABataJCQAAAA1JREFUGFdjYGBgYAAAAAUAAYoz4wAAAAAASUVORK5CYII=
+</value>
+  </data>
+  <data name="picPortrait.Image" type="System.Drawing.Bitmap, System.Drawing" mimetype="application/x-microsoft.net.object.bytearray.base64">
+    <value>
+        iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAABGdBTUEAALGPC/xhBQAAAAlwSFlzAAAO
+        wAAADsABataJCQAAAA1JREFUGFdjYGBgYAAAAAUAAYoz4wAAAAAASUVORK5CYII=
+</value>
+  </data>
```
Comment (ekolis) [inline FrEee.UI.WinForms/Forms/VehicleDesignForm.resx:132]: Where did these come from? Do the picture boxes have default images now?

File: FrEee.UI.WinForms/Forms/EmpireSetupForm.resx (hunk 3 above)
Comment (unknown555) [reply FrEee.UI.WinForms/Forms/EmpireSetupForm.resx:150]: I don't recall doing anything with resource files and I don't know how they work.  Maybe Visual Studio did this automatically?

Comment (ekolis) [reply FrEee.UI.WinForms/Forms/EmpireSetupForm.resx:150]: OK, I guess it's fine, then. Visual Studio is weird sometimes.

File: FrEee.UI.WinForms/Forms/VehicleDesignForm.resx (hunk 4 above)
Comment (ekolis) [reply FrEee.UI.WinForms/Forms/VehicleDesignForm.resx:132]: Probably done by Visual Studio as well.

File: FrEee.UI.WinForms/Forms/EmpireSetupForm.cs (hunk 1 above)
Comment (unknown555) [reply FrEee.UI.WinForms/Forms/EmpireSetupForm.cs]: I've deleted the two functions in the new commit just now.

**Comment by ekolis** [comment]
Thanks for the pull request! 🙂

**Comment by ekolis** [comment]
This fixes #280

**Pull request merged**